### PR TITLE
Bug 1686476: Force http1.1 in OAuth server

### PR DESCRIPTION
**Pull request by enj**
@openshift/sig-auth @deads2k @smarterclayton @ironcladlou @Miciah @jupierce

### CODE REVIEW

**Comment by enj** [comment]
@knobunc

File: test/extended/oauth/http1.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	o "github.com/onsi/gomega"
+	"golang.org/x/net/http2"
+
+	"k8s.io/client-go/rest"
+
+	"github.com/openshift/origin/pkg/oauth/util"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("The OAuth server", func() {
```
Comment (deads2k) [inline test/extended/oauth/http1.go]: not sure this is the right name format

Comment (enj) [reply test/extended/oauth/http1.go]: (discussed in-person, will fix later as needed)

**Pull request merged**